website/packages/docs/src/pages/pkg-webpack-loader.mdx

```diff
@@ -307,3 +307,18 @@ See [Shorthand properties](/shorthand) for more information.
 
 - Type: `boolean`
```

Comment:
ah this is a small nitpick, but can we ensure the `Type: boolean` `Default: false` stuff is still placed at the end of the `sortShorthand` subsection?